server/sonar-web/src/main/js/apps/projects-admin/ChangeVisibilityForm.js

```diff
@@ -23,40 +23,44 @@ import Modal from 'react-modal';
 import classNames from 'classnames';
 import UpgradeOrganizationBox from '../../components/common/UpgradeOrganizationBox';
 import { translate } from '../../helpers/l10n';
-import type { Organization } from '../../store/organizations/duck';
+/*:: import type { Organization } from '../../store/organizations/duck'; */
 
+/*::
 type Props = {
   onClose: () => void,
   onConfirm: string => void,
   organization: Organization
 };
+*/
 
+/*::
 type State = {
   visibility: string
 };
+*/
 
 export default class ChangeVisibilityForm extends React.PureComponent {
-  props: Props;
-  state: State;
+  /*:: props: Props; */
+  /*:: state: State; */
 
-  constructor(props: Props) {
+  constructor(props /*: Props */) {
     super(props);
     this.state = { visibility: props.organization.projectVisibility };
   }
 
-  handleCancelClick = (event: Event) => {
+  handleCancelClick = (event /*: Event */) => {
     event.preventDefault();
     this.props.onClose();
   };
 
-  handleConfirmClick = (event: Event) => {
+  handleConfirmClick = (event /*: Event */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)